pageview_digest/wsgi.py

```diff
@@ -88,8 +88,9 @@ def application(env, start_response):
     if path == "/trending.json" and "site" in params:
         try:
             site = params.get("site", [None])[0]
-            offset = params.get("offset", [DEFAULT_OFFSET])[0]
-            limit = params.get("limit", [DEFAULT_LIMIT])[0]
+            offset = int(params.get("offset", [DEFAULT_OFFSET])[0])
```

Comment:
what if the offset param is provided, but empty?
might end up with an `IndexError`